File: app/src/main/java/com/artmurka/artmurkaapp/model/pojo/itemlist/formorder/Success.java
package com.artmurka.artmurkaapp.model.pojo.itemlist.formorder;


import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Success {


    @SerializedName("msg")
    @Expose
    private final ThreadLocal<String> msg = new ThreadLocal<>();

    public String getMsg() {
        return msg.get();
    }

    public void setMsg(String msg) {
        this.msg.set(msg);
    }
}

File: app/src/main/java/com/artmurka/artmurkaapp/model/pojo/itemlist/categories/Success.kt
package com.artmurka.artmurkaapp.model.pojo.itemlist.categories


import android.os.Parcelable
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
class Success  : Parcelable {

    @SerializedName("cat_img")
    @Expose
    var catImg: String? = null
    @SerializedName("childs")
    @Expose
    var childs: List<Success>? = null
    @SerializedName("cat_descr")
    @Expose
    var catDescr: String? = null

    @SerializedName("goods_count")
    @Expose
    var goodsCount: Int = 0
    @SerializedName("cat_id")
    @Expose
    var catId: String? = null
    @SerializedName("cat_name")
    @Expose
    var catName: String? = null
    @SerializedName("cat_url")
    @Expose
    var catUrl: String? = null

}